corehq/apps/translations/app_translations/upload_app.py

```diff
@@ -52,61 +49,89 @@ def run_translation_checker(file_obj):
         result_wb = validate_workbook(file_obj, translation_checker_messages)
     except FatalError as e:
         translation_checker_messages.append(
-            _("Workbook check failed to finish due to the following error : %s" % e))
+            _("Workbook check failed to finish due to the following error : %s" % e)
+        )
     return translation_checker_messages, result_wb
 
 
-def validate_bulk_app_translation_upload(app, workbook, email, lang_to_compare, file_obj):
+def validate_bulk_app_translation_upload(
+    app, workbook, email, lang_to_compare, file_obj
+):
```

Comment:
I think this might be an overkill for indentation. And possibly for some other changes in the file as well.

Also, this is stretching to places outside what you are covering in this PR.

For both these points, sharing this old PR where i tried something similar and got some feedback on it, mainly
https://github.com/dimagi/commcare-hq/pull/27769#discussion_r438407896
Please don't let this stop you from making improvements to the code base where you find necessary.

I would sugggest dropping this commit using interactive rebase and limit this to changes you think are worth keeping for the code you touched. happy to talk more in detail